lib/fastlane/plugin/wpmreleasetoolkit/actions/configure/configure_apply_action.rb

```diff
@@ -10,6 +10,17 @@ module Fastlane
   module Actions
     class ConfigureApplyAction < Action
       def self.run(params = {})
+
+        ### Make sure secrets repo is at the proper hash as specified in .configure
+        repo_branch_name = Fastlane::Helper::ConfigureHelper.repo_branch_name
+        file_branch_name = Fastlane::Helper::ConfigureHelper.configure_file_branch_name
+        repo_hash = Fastlane::Helper::ConfigureHelper.repo_commit_hash
+        file_hash = Fastlane::Helper::ConfigureHelper.configure_file_commit_hash
+
+        unless repo_branch_name == file_branch_name && repo_hash == file_hash
+          sh("cd #{repository_path} && git checkout #{file_hash}")
```

Comment:
In addition to checking out the hash, should we also do a `git fetch` here? Its possible that the remote ref won't be available.

Reply:
I think we should, too!  Good catch - will add that!

Reply:
Added it!